src/main/cli/settings.c

```diff
@@ -1131,7 +1131,11 @@ const clivalue_t valueTable[] = {
 
     { "angle_level_strength",       VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].P) },
     { "horizon_level_strength",     VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].I) },
-    { "horizon_transition",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].D) },
+    { "angle_level_derivative_gain",VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].D) },
+    { "horizon_transition",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, horizonTransition) },
+    { PARAM_NAME_ANGLE_FEEDFORWARD, VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].F) },
+    { PARAM_NAME_LEVEL_GYRO_FF_SCALE, VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_PID_PROFILE, offsetof(pidProfile_t, levelGyroFeedforwardScale) },
+    { "angle_dterm_lpf_hz",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 1000 }, PG_PID_PROFILE, offsetof(pidProfile_t, angle_dterm_lpf_hz) },
```

Comment:
Let's please be consistent in the naming of all parameters associated with angle mode.  

Even though the mode is 'angle mode' I think it will be simpler to start all of these CLI values with the text 'level' and would be grateful if you could do that.

Otherwise start them all with angle, and drop the level.

Reply:
Renaming things is often a difficult business. I'm happy to name things however the project team decides, however, this angle vs level business needs to be sorted out and agreed before we change the code again. Some terms have now been changed from angle to level and back again! 🤣 And now we want them to be changed BACK to level!